cmd/tools/gendynamicconfig/dynamic_config.tmpl

```diff
@@ -0,0 +1,167 @@
+{{- /*gotype: go.temporal.io/server/cmd/tools/gendynamicconfig.dynamicConfigData*/ -}}
```

Comment:
This line enables intellisense in GoLand.

Reply:
ooh, very cool. I'm jealous of that, the template has been frustrating to edit